pkg/config/services.go

```diff
@@ -92,8 +92,10 @@ var SupportedServices = serviceConfigs{
 			aws.String("apigateway"),
 		},
 		DimensionRegexps: []*regexp.Regexp{
-			regexp.MustCompile("apis/(?P<ApiName>[^/]+)$"),
-			regexp.MustCompile("apis/(?P<ApiName>[^/]+)/stages/(?P<Stage>[^/]+)$"),
+			regexp.MustCompile("restapis/(?P<ApiName>[^/]+)$"),
+			regexp.MustCompile("restapis/(?P<ApiName>[^/]+)/stages/(?P<Stage>[^/]+)$"),
+			regexp.MustCompile("apis/(?P<ApiId>[^/]+)$"),
+			regexp.MustCompile("apis/(?P<ApiId>[^/]+)/stages/(?P<Stage>[^/]+)$"),
```

Comment:
Can you please add a comment to mark which regexes are for GW v1 vs v2?

Reply:
Done 👍